### PR TITLE
Add a new value inner-spin-button to -moz-appearance in stylo.

### PR DESCRIPTION
**Pull request by ghoo1125**
Add a new value inner-spin-button to -moz-appearance. See bug 1417751.
https://reviewboard.mozilla.org/r/200244/diff/6#index_header

---

- [ ] `./mach build -d` does not report any errors
- [ ] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because _____

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/19406)

### CODE REVIEW

**Comment by aethanyc** [comment]
@bors-servo r=heycam
It was reviewed in https://bugzilla.mozilla.org/show_bug.cgi?id=1421161#c3

File: components/style/properties/longhand/box.mako.rs [hunk 1]
```diff
@@ -817,8 +817,8 @@ ${helpers.single_keyword("-moz-appearance",
                             scrollbar-horizontal scrollbar-small scrollbar-vertical scrollbarbutton-down
                             scrollbarbutton-left scrollbarbutton-right scrollbarbutton-up scrollbarthumb-horizontal
                             scrollbarthumb-vertical scrollbartrack-horizontal scrollbartrack-vertical searchfield
-                            separator spinner spinner-downbutton spinner-textfield spinner-upbutton splitter statusbar
-                            statusbarpanel tab tabpanel tabpanels tab-scroll-arrow-back tab-scroll-arrow-forward
+                            separator inner-spin-button spinner spinner-downbutton spinner-textfield spinner-upbutton splitter
```
Comment (emilio) [inline components/style/properties/longhand/box.mako.rs]: These were ordered, any chance to keep them that way? Also, this probably needs a bindings update.

**Review by emilio — COMMENTED** [review]
Looks fine. I can help out with the bindings update if you want, just ping me on IRC :)

**Comment by emilio** [comment]
Oh ok, was already reviewed... As long as it builds (though I don't think it would)

File: components/style/properties/longhand/box.mako.rs (hunk 1 above)
Comment (ghoo1125) [reply components/style/properties/longhand/box.mako.rs]: Yep, my mistake. I have fixed it.

**Comment by ghoo1125** [comment]
@bors-servo  r=heycam
I have fixed the order.

**Comment by KiChjang** [comment]
```
error[E0425]: cannot find value `ThemeWidgetType_NS_THEME_INNER_SPIN_BUTTON` in module `structs`
     --> /Users/servo/buildbot/slave/mac-dev-unit/build/target/geckolib/debug/build/style-abfaa561961fdc53/out/gecko_properties.rs:14460:30
      |
14460 |                     structs::ThemeWidgetType_NS_THEME_INNER_SPIN_BUTTON as u8,
      |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ did you mean `ThemeWidgetType_NS_THEME_SPINNER_DOWNBUTTON`?

error[E0425]: cannot find value `ThemeWidgetType_NS_THEME_INNER_SPIN_BUTTON` in module `structs`
     --> /Users/servo/buildbot/slave/mac-dev-unit/build/target/geckolib/debug/build/style-abfaa561961fdc53/out/gecko_properties.rs:14741:22
      |
14741 |             structs::ThemeWidgetType_NS_THEME_INNER_SPIN_BUTTON as u8;
      |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ did you mean `ThemeWidgetType_NS_THEME_SPINNER_DOWNBUTTON`?
```

**Comment by ghoo1125** [comment]
@bors-servo  r=heycam
Update Gecko bindings.

**Comment by ghoo1125** [comment]
@bors-servo r=heycam
Fix tidy test which line is over 120 characters...

**Comment by emilio** [comment]
Err,

@bors-servo retry

**Comment by ghoo1125** [comment]
Hi @emilio , thanks for helping this out :)

**Pull request merged**